### ISSUE TITLE
create_runtime_policy.sh script ignores excluded paths

### ISSUE THREAD

**Issue by gnurugs** [keylime/keylime#1706]
# Environment
 * OS / version: Ubuntu 24.04.1 LTS
 * Processor architecture: x86_64
 * TPM Manufacturer: 
 * Keylime version: v7.11.0

# Description 
The keylime docs for Runtime Integrity Monitoring documents the use of `create_runtime_policy.sh` script to generate runtime policy. The script has an optional `-e` argument that takes a file containing a list of paths to be excluded in the runtime policy. However, using this script with -e flag does not populate the exclude list in the policy. 

looking at the code for this script it uses `keylime/cmd/convert_runtime_policy.py` which takes in allowlists and excludelist to generate the runtime policy, so I mannually tested the allowlist generated by `create_runtime_policy.sh` script and excludelist to generate a policy which did include the excludelist.

## Expected behavior
exludelist to be populated in the runtime policy generated by using  `create_runtime_policy.sh` script

## Actual behavior 
snippet of runtime_policy.json 

    {"meta":...}, "excludes": [], "keyrings": {}, "ima": {"ignored_keyrings": [], "log_hash_alg": "sha1", "dm_policy": null}, "ima-buf": {}, "verification-keys": ""}

Notice `excludes` is empty.

## Steps to reproduce problem
In Keylime agent environment run  `create_runtime_policy.sh` script with `-e` flag:

    keylime/scripts/create_runtime_policy.sh -a sha256sum -e excludelist.txt -o runtime_policy.json

**Comment by THS-on** [comment]
Not sure why this happens. How does your `excludelist.txt` look like? @sergio-correia can you have a closer look at this issue?

Note we are also currently in the process of deprecating some of the scripts in favor of the new `keylime-policy` tool.

**Comment by sergio-correia** [comment]
@gnurugs: as @THS-on suggested, please share the contents of the `excludelist.txt` file so we can investigate the issue.

**Comment by gnurugs** [comment]
my `excludelist.txt` looks like this:

    cat excludelist.txt
    /tmp/.*
    /var/cache/.*
    /var/lock/.*
    /var/log/.*
    /var/run/.*
    /var/spool/.*
    /var/tmp/.
    /var/lib/dpkg/*

this `excludelist.txt` works as expected with the script `keylime/cmd/convert_runtime_policy.py`

**Comment by sergio-correia** [comment]
Yeah, the file looks good. Could you please try to edit the `keylime/scripts/create_runtime_policy.sh` file in the following way, and test if it helps?

Replace line 272 (https://github.com/keylime/keylime/blob/master/scripts/create_runtime_policy.sh#L272) with this:
```
[ -f $EXCLUDE_LIST ] && CONVERT_CMD_OPTS="$CONVERT_CMD_OPTS --excludelist "$(readlink -f -- $EXCLUDE_LIST)""
```

If possible, please try out also the `keylime-policy` tool.

**Comment by gnurugs** [comment]
@sergio-correia, That fixed the issue. Thanks for checking this. 

@ansasaki, It is lot easier to use `keylime-policy` tool. Thanks for including `keylime-policy` tool in the documentation.

**Comment by ansasaki** [comment]
Fixed by #1714
